<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenEditTab</class>
 <widget class="QWidget" name="ScreenEditTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="screen_attribute_hbox" stretch="0,1,0,1">
     <item>
      <widget class="QLabel" name="screen_type_label">
       <property name="text">
        <string>Screen type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="screen_type_combo"/>
     </item>
     <item>
      <widget class="QLabel" name="resource_id_label">
       <property name="text">
        <string>Resource ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resource_id_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="control_separator_line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="font_control_hbox" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QPushButton" name="bold_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Bold</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="italic_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Italic</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="underline_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Underline</string>
       </property>
       <property name="text">
        <string>U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="text_color_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Insert Text Color Tag</string>
       </property>
       <property name="text">
        <string>Text Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="text_color_combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Selected Color Tag</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="font_control_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="screen_text_edit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
